### ISSUE TITLE
1.15.2: 9.0.0-BETA2 Slight documentation error

### ISSUE THREAD

**Issue by JokerBoss99** [kangarko/ChatControl-Pro#1715]
Hello, I encountered a slight documentation error under the Grammar section in settings.yml
The error is linked below, but in the settings.yml it says:
Grammar:
  # Punctualize every sentence (insert a dot at the end) longer than the given letters.
  # Set to 0% to disable.
  Insert_Dot_Message_Length: 0

Which isn't true, because the error I received told me that it had to be an integer, not a string.

Basically, putting it to 0 makes it work, 0% like the documentation says doesn't work.
[error.txt](https://github.com/kangarko/ChatControl-Pro/files/4780642/error.txt)

**Comment by lackoSK** [comment]
Hello,

Sorted. Wait till the next update. 

Sincerely,
Ladislav from MineAcademy.org
